src/test/stub/hashset.c

```diff
@@ -0,0 +1,125 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+#include <stdint.h>
+#include <assert.h>
+#include <stdlib.h>
+
+/*
```

Comment:
Link to issue tracking why we can't do this with unbounded arrays. There should be a diffblue issue too